lib/endpoint/find-parameters.js

```diff
@@ -216,3 +217,11 @@ function addPaginationParameters (state) {
     result.params.push(...PAGINATION_VARIABLES)
   })
 }
+
+function makeRelativeLinksAbsolute (state) {
+  const results = state.results[0].params
```

Comment:
You now iterate only trough the params of the first results. You have to iterate trough all results, too.

Reply:
ok let me update that